include/uxr/agent/message/InputMessage.hpp

```diff
@@ -100,6 +102,22 @@ template bool InputMessage::get_payload(dds::xrce::WRITE_DATA_Payload_Data& data
 template bool InputMessage::get_payload(dds::xrce::HEARTBEAT_Payload& data);
 template bool InputMessage::get_payload(dds::xrce::ACKNACK_Payload& data);
 
+inline bool InputMessage::get_raw_payload(uint8_t* buf, size_t len)
+{
+    bool rv = false;
+    if (subheader_.submessage_length() <= len)
+    {
+        deserializer_.deserializeArray(buf, subheader_.submessage_length(), fastcdr::Cdr::BIG_ENDIANNESS);
+        rv = true;
+    }
+    return rv;
+}
+
+inline bool InputMessage::jump_payload()
+{
+    return deserializer_.jump(size_t(subheader_.submessage_length()));
```

Comment:
Check if `jump` could launch an exception.

Reply:
No, it returns a boolean.